packages/gitbeaker-core/src/services/ContainerRegistry.ts

```diff
@@ -32,13 +32,13 @@ export class ContainerRegistry extends BaseService {
   removeTags(
     projectId: string | number,
     repositoryId: number,
-    tagNameRegex: string,
+    nameRegex: string,
     options?: Sudo & { keepN: string; olderThan: string },
```

Comment:
This should work until gitlab removes the deprecated `name_regex`. Just ignore the existence of `name_regex_delete` for now.
```suggestion
    options?: Sudo & { nameRegexKeep: string, keepN: string; olderThan: string },
```